pinot-common/src/main/java/org/apache/pinot/common/metrics/ControllerMetrics.java

```diff
@@ -25,8 +25,18 @@
  * Metrics for the controller.
  */
 public class ControllerMetrics extends AbstractMetrics<AbstractMetrics.QueryPhase, ControllerMeter, ControllerGauge, ControllerTimer> {
+
+  // FYI this is incorrect as it generate metrics named without a dot after pinot.controller part,
+  // but we keep this default for backward compatibility in case someone relies on this format
+  // see Servermetrics or BrokerMetrics class for correct prefix should be use
+  public static final String METRICS_PREFIX_DEFAULT = "pinot.controller";
```

Comment:
Missing "."

Reply:
see the comment, it is mostly for backward compatible as the old prefix don't have the-dot

Reply:
thanks